tests/framework/clients/rancher/client.go

```diff
@@ -157,6 +149,19 @@ func (c *Client) WithSession(session *session.Session) (*Client, error) {
 	return NewClient(c.restConfig.BearerToken, session)
 }
 
+// Catalog is a function that takes a clusterID and instanitates a catalog client to directly communicate with that specific cluster.
+func (c *Client) Catalog(clusterID string) (*catalog.Client, error) {
```

Comment:
I would keep the original client incase we want to install charts on local.  This could be renamed "GetClusterCatalogClient"